### ISSUE TITLE
[Request] Include visual preview of starters

### ISSUE THREAD

**Issue by Hahlh** [maizzle/docs#123]
Good day! :)

Thanks for this incredible project. I am super excited to get started with this and use it for creating a well polished newsletter.

It's a little difficult atm to decide which starter to go for, because I would need to test them to know what they are. 
It would be great to have additional visual information in the docs and the individual repos that show what the starter is about.

Thank you for your work! :sunny:

**Comment by cossssmin** [comment]
Good call, some starters do have screenshots but some don't and the docs individual pages don't show them either.

**Comment by skchronicles** [comment]
Hey @cossssmin,

Thank you for creating and maintaining this useful project. I agree with @Hahlh. Would it be possible to add a link to a pre-built starter page? I see there are screen shots, but they are pretty small so it is hard to get a feel for a given starter.

I think including a link to something that was built with maizzle would lead to better adoption, and it would help a user that is interested in using a starter to make a faster, more informed decision.

**Comment by cossssmin** [comment]
Yeah, next up after the 4.0 release will be docs improvement for sure 👍
